src/ProgressBar.js

```diff
@@ -58,7 +52,14 @@ const propTypes = {
   label: PropTypes.node,
   srOnly: PropTypes.bool,
   striped: PropTypes.bool,
-  active: PropTypes.bool,
+  animated: PropTypes.bool,
```

Comment:
I know a bunch of comments are missing, but we want to try and add/backfill docs for any of the components we're working on. Lets add descriptive comments 

(you don't need the the type doclet for everyone btw, docs will infer it from the propType, we use it as an override like in variant below)